src/System.Text.Utf8String/System/Text/Utf8String.cs

```diff
@@ -587,6 +587,43 @@ public Utf8String TrimStart(byte[] trimCodeUnits)
             throw new NotImplementedException();
         }
 
+        public Utf8String TrimStart(Utf8String prefix)
```

Comment:
"Prefix" isn't very semantically correct, true. 
I considered a managed string's TrimStart() as the canonical intended functionality. Wouldn't a) cause potential confusion between `string` and `Utf8String` 's use?